dcos/resource_dcos_package.go

```diff
@@ -59,7 +62,7 @@ func resourceDcosPackage() *schema.Resource {
 			"sdk": {
 				Type:        schema.TypeBool,
 				Optional:    true,
-				Default:     false,
+				Default:     true,
```

Comment:
I don't think this is a good idea to be set by default. If being set on non SDK packages errors will occur. At the moment we tent to leave it to false

Reply:
Reverted to false